CHANGELOG.md

```diff
@@ -2,11 +2,15 @@
 
 **Breaking changes**
 
+* supvervisord collector reports "start_time_seconds" rather than "uptime"
```

Comment:
Should we also add this to the recording rules file for backward compatibility?

Reply:
Done